src/FMBot.Bot/SlashCommands/TopSlashCommands.cs

```diff
@@ -53,7 +54,8 @@ public class TopSlashCommands : InteractionModuleBase
         [Summary("Time-period", "Time period")][Autocomplete(typeof(DateTimeAutoComplete))] string timePeriod = null,
         [Summary("Billboard", "Show top artists billboard-style")] bool billboard = false,
         [Summary("User", "The user to show (defaults to self)")] string user = null,
-        [Summary("XXL", "Show extra top artists")] bool extraLarge = false,
+        [Summary("XXL", "Show more top artists")] bool extraLarge = false,
+        [Summary("XXS", "Show less top artists")] bool extraSmall = false,
```

Comment:
Instead of using two options you can also allow users to pick the enum in slash commands. There are some other slash commands that have enums, you can use those as example.